test/plugin/scenarios/armeria-0.96plus-scenario/support-version.list

```diff
@@ -14,5 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+0.98.0
```

Comment:
@wu-sheng let me add this version to verify it works (although I verified locally), and revert this to merge #4328 :)

Reply:
Make sense to me.